core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -192,11 +215,15 @@
           </div>
         </li>
       </ul>
-      <hint-and-solution-buttons ng-if="windowIsNarrow"></hint-and-solution-buttons>
+      <hint-and-solution-buttons ng-if="windowIsNarrow || getNumberOfCorrectAnswers()"></hint-and-solution-buttons>
```

Comment:
getNumberOfCorrectAnswers() > 0 (just to make it more clear)

Reply:
Done!